node/src/components/storage.rs

```diff
@@ -1153,6 +1173,29 @@ fn insert_to_deploy_index(
     Ok(())
 }
 
+fn should_move_storage_files_to_network_subdir(root: &PathBuf, file_names: &[&str]) -> bool {
+    file_names
+        .iter()
+        .map(|file_name| root.join(file_name))
+        .all(|file_path| file_path.exists())
+}
+
+fn move_storage_files_to_network_subdir(
+    root: &PathBuf,
+    subdir: &PathBuf,
+    file_names: &[&str],
+) -> Result<(), Error> {
+    file_names
+        .iter()
+        .map(|file_name| (root.join(file_name), subdir.join(file_name)))
+        .map(|(src, dest)| {
+            fs::rename(src.clone(), dest.clone()).map_err(|_| Error::UnableToMoveFile(src, dest))
```

Comment:
Note: I actually do want to preserve the original fs error here, I could use some advice on what the cleanest way to get that into the Thiserror enum.